sdl2_image-config.cmake.in

```diff
@@ -0,0 +1,96 @@
+# sdl2_image cmake project-config input for ./configure scripts
+
+set(SDL2_image_FOUND TRUE)
+
+set(SDL2IMAGE_AVIF  @LOAD_AVIF@)
```

Comment:
Can the substituted variables be 1 and 0, or do they need to be TRUE/FALSE, or ON/OFF?

Reply:
CMake does not care between true/false, TRUE/FALSE, on/off, or ON/OFF

https://cmake.org/cmake/help/latest/command/if.html#basic-expressions
> True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number (including floating point numbers). False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. Named boolean constants are case-insensitive. If the argument is not one of these specific constants, it is treated as a variable or string (see [Variable Expansion](https://cmake.org/cmake/help/latest/command/if.html#variable-expansion) further below) and one of the following two forms applies.